package.json

```diff
@@ -2,7 +2,7 @@
   "name": "esp-idf-extension",
   "displayName": "ESP-IDF",
   "description": "Develop and debug applications for Espressif ESP32, ESP32-S2 chips with ESP-IDF",
-  "version": "1.6.5",
+  "version": "1.6.6",
```

Comment:
Why do you update the package version ? Sorry but we can't update the package version this way.